src/grid/style.scss

```diff
@@ -145,3 +145,57 @@
 		color: var(--stc-color-white);
 	}
 }
+
+.stc-grid-new-item .stc-grid-item-header {
+    padding: 20px 20px;
+    border-top: 1px solid #e9e9e9;
+    background: #FFFFFF;
+    border: 1px solid #D1D5DB;
```

Comment:
Please use css variables here and wherever required. 
Variables are declared in this file - src/style.scss